### ISSUE TITLE
Unexpected behaviour on get method

### ISSUE THREAD

**Issue by harukaze-sm** [ps73/feathers-prisma#6]
### Steps to reproduce

- Tried to use feathers-casl with prisma adapter, but setting permissions to specific issues gives the following error
```javascript
{
    "name": "NotFound",
    "message": "No record found for id 'ckzqtphw00000hz3x1g9szk3f' and query.id 'ckzqtphw00000hz3x1g9szk3f'",
    "code": 404,
    "className": "not-found",
    "errors": {}
}
```
- simple example to reproduce this issue
```javascript
// abilities.ts https://feathers-casl.netlify.app/getting-started.html#about for better reference 
import { createAliasResolver, makeAbilityFromRules } from 'feathers-casl';
import { AbilityBuilder, Ability } from '@casl/ability';
import { User } from '@prisma/client';

export const defineRulesFor = (user: User) => {
  const { can, rules } = new AbilityBuilder(Ability);

  can('read', 'users', { id: user.id }); # note that { id: user.id } causes this issue
  can('update', 'users', { id: user.id });
  can('delete', 'users', { id: user.id });

  return rules;
};
```

```javascript
import { authorize } from 'feathers-casl/dist';

 before: {
    all: [authenticate('jwt'), loadAbilities()], // load abilities are basically same as https://feathers-casl.netlify.app/getting-started.html#add-abilities-to-hooks-context
    get: [authorize()], // this hook causes the JSON response written at the top
  },
```

### Expected behavior

It should allow user to get only info related to their id, and give 403 on any other user

### Actual behavior

<img width="839" alt="image" src="https://user-images.githubusercontent.com/56676964/154469024-920fe883-c397-4214-9422-dbdf3ea865a7.png">

### System details

Macbook Pro M1 13inch
MacOS Monterey 12.2

**Module versions** (especially the part that's not working):
```sh
feathers-prisma@0.5.6
feathers-casl@0.7.1
```

**NodeJS version**:
```sh
v16.13.0
```

**Operating System**:
```sh
MacOS
```

**Comment by harukaze-sm** [comment]
Update:

Found that after removing checkIdInQuery function inside _get method helped my resolve my issue, works as excepted

<img width="839" alt="image" src="https://user-images.githubusercontent.com/56676964/154475888-4c2fa0de-c512-4cc4-9a27-d08faa3322bb.png">

**Comment by ps73** [comment]
Just for documentation. It is the same wrong behavior in _remove and _update method if we use feathers-casl.

**Comment by ps73** [comment]
Issues were fixed this in v0.5.7
